src/project/inotationproject.h

```diff
@@ -51,6 +51,9 @@ class INotationProject
     virtual const CloudProjectInfo& cloudInfo() const = 0;
     virtual void setCloudInfo(const CloudProjectInfo& info) = 0;
 
+    virtual ExportInfo exportInfo() const = 0;
```

Comment:
I think we can store this information in ExportProjectScenario instead of NotationProject
And it might be like 
std::map<path /* projectPath*/, ExportInfo>